client/src/index.js

```diff
@@ -41,6 +41,9 @@ const { publicClient } = configureChains(
   ]
 );
 
+export const alchemyMainnetEthersProvider =
```

Comment:
Can we reuse the alchemy provider here instead of creating a new one? https://github.com/zkp2p/zk-p2p/blob/fad784cc738ae275291504905ed3c0fb64591b86/client/src/index.js#L39

Reply:
@asoong we can't because the other one is linked to wagmi and is configured to work only on base